tools/tsdb-chunks/main.go

```diff
@@ -69,16 +69,22 @@ func printChunksFile(filename string, printSamples bool) error {
 
 			it := c.Iterator(nil)
 			six := 0
-			for it.Err() == nil && it.Next() {
-				ts, val := it.At()
-				if ts < minTS {
-					minTS = ts
+			for valType := it.Next(); it.Err() == nil && valType != chunkenc.ValNone; valType = it.Next() {
```

Comment:
[nit] By contract, `Next()` must return `ValNone` on error.

```suggestion
			for valType := it.Next(); valType != chunkenc.ValNone; valType = it.Next() {
```

Reply:
i thought of this, but wasn't sure, fixed